src/main/webapp/jsp/instructorComments.jsp

```diff
@@ -8,6 +8,9 @@
     <link type="text/css" href="/stylesheets/omniComment.css" rel="stylesheet">
 </c:set>
 <c:set var="jsIncludes">
+    <script type="text/javascript" src="/js/lib/tinymce.min.js"></script>
+    <script type="text/javascript" src="/js/richTextEditor.js"></script>
+    <script type="text/javascript" src="/js/additionalQuestionInfo.js"></script>
```

Comment:
Looks like I missed this :p this was removed in one of my previous PRs so it should not be re-added, however it is not show-stopping even if present so it is fine.